samples/react/server/server.js

```diff
@@ -58,15 +56,21 @@ export function renderView(callback, path, data, viewBag) {
       App Rendering
     */
     initializei18n(state)
-      .then(() =>
+      .then(() => {
         // renderToStringWithData() allows any GraphQL queries to complete their async call
         // before the SSR result is returned, so that the resulting HTML from GQL query results
         // is included in the SSR'ed markup instead of whatever the 'loading' state is.
         // Not using GraphQL? Use ReactDOMServer.renderToString() instead.
-        renderToStringWithData(
-          <AppRoot path={path} Router={StaticRouter} graphQLClient={graphQLClient} />
+        const renderedAppHtml = renderToStringWithData(
+          <AppRoot path={path}
+            Router={StaticRouter}
+            graphQLClient={graphQLClient}
+            ssrState={state}
+          />
         )
-      )
+
+        return renderedAppHtml;
```

Comment:
We could get rid of the `{ }` and `return` statement in this part of the promise chain now that this clause has been simplified.